polygeist/tools/cgeist/Test/Verification/sycl/structvec.cpp

```diff
@@ -1,7 +1,8 @@
 // RUN: clang++ -fsycl -fsycl-device-only -O0 -w -emit-mlir %s -o - | FileCheck %s
+// XFAIL: *
```

Comment:
Opened issue https://github.com/intel/llvm/issues/7800 to track this problem. The test case failed for any pointer type, except for `char*`. Now that we use a memref to represent `char*` (like we do for short*, ...) the existing issue is exposed.